src/js/disability-benefits/526EZ/config/form.js

```diff
@@ -122,58 +130,63 @@ const formConfig = {
-        if (isReviewPage) {
-          return baseString.replace('Review ', '');
-        }
-        return baseString;
-      },
+    veteranDetails: {
+      title: (isReviewPage) => `${isReviewPage ? 'Review ' : ''}Veteran Details`,
       pages: {
-        veteranInformation: createVerifiedVeteranInfoPage(fullSchema526EZ),
-        specialCircumstances: { // TODO: create page file and reuse 
-          title: 'Special Circumstances',
-          path: 'review-veteran-details/special-circumstances',
-          depends: formData => formData.prefilled,
+        veteranInformation: {
+          title: 'Veteran Information', // TODO: Figure out if this is even necessary
+          path: 'veteran-information',
           uiSchema: {
-            'ui:description': specialCircumstancesDescription,
-            'view:suicidal': {
-              'ui:title': 'In crisis or thinking of suicide?'
-            },
-            'view:homeless': {
-              'ui:title': 'Homeless or at risk of becoming homeless?'
+            'ui:field': ReviewCardField,
+            'ui:title': 'Veteran information',
+            'ui:options': {
+              viewComponent: veteranInformationViewField
             },
-            'view:extremeFinancialHardship': {
-              'ui:title': 'Suffering from extreme financial hardship?'
+            fullName: fullNameUI,
+            socialSecurityNumber: merge({}, ssnUI, {
+              'ui:title': 'Social Security number (must have this or a VA file number)',
+              'ui:required': form => !form.vaFileNumber
+            }),
+            vaFileNumber: {
+              'ui:title': 'VA file number (must have this or a Social Security number)',
+              'ui:required': form => !form.socialSecurityNumber,
+              'ui:help': VAFileNumberDescription,
+              'ui:errorMessages': {
+                pattern: 'Your VA file number must be between 7 to 9 digits'
+              }
             },
-            'view:blindOrSightImpaired': {
-              'ui:title': 'Blind or sight-impaired?'
+            dateOfBirth: currentOrPastDateUI('Date of birth'),
+            gender: {
+              'ui:title': 'Gender',
+              'ui:options': {
+                labels: genderLabels
+              }
             }
           },
           schema: {
             type: 'object',
+            required: ['fullName', 'gender', 'dateOfBirth'],
             properties: {
-              'view:suicidal': {
-                type: 'boolean'
+              fullName,
+              socialSecurityNumber: {
+                type: 'string'
               },
-              'view:homeless': {
-                type: 'boolean'
+              vaFileNumber: {
+                type: 'string',
+                pattern: '^[cC]{0,1}\\d{7,9}$'
               },
-              'view:extremeFinancialHardship': {
-                type: 'boolean'
+              gender: {
+                type: 'string',
+                'enum': ['F', 'M']
```

Comment:
Are you planning on pulling these fields from the schema later?

Reply:
Yeah, definitely. For now, I copied over what we were already using in `createVerifiedVeteranInfoPage()`.

I don't remember offhand if much of this information is even _in_ the schema, now that I think about it... :thinking: That's a larger conversation, though. :stuck_out_tongue: